cmd/system-probe/subcommands/run/command.go

```diff
@@ -295,3 +298,25 @@ func setupInternalProfiling(cfg ddconfig.ConfigReader, configPrefix string, log
 func isValidPort(port int) bool {
 	return port > 0 && port < 65536
 }
+
+func logUserAndGroupID(log log.Component) {
+	resolver, err := usergroup.NewResolver()
+	if err != nil {
+		log.Warn("cannot create user/group resolver")
+	} else {
+		uid := os.Getuid()
+		gid := os.Getgid()
+		userName, err := resolver.ResolveUser(uid)
```

Comment:
You should not use the resolver from the security side here. It only exists on linux for now, and this is why your PR is failing the linter job on windows

Reply:
you should most likely use https://pkg.go.dev/os/user#Current instead here